src/App.js

```diff
@@ -1,29 +1,42 @@
-import logo from './logo.svg';
+import {
+  createBrowserRouter,
+  RouterProvider,
+  Route,
+  createRoutesFromElements,
+} from 'react-router-dom';
+// import { ToastContainer } from 'react-toastify';
+import Reserve from './components/Reserve';
+import Doctors from './components/Doctors';
+import Doctor from './components/Doctor';
 import './App.css';
+import SignUp from './components/auth_pages/Signup';
+import Login from './components/auth_pages/Login';
+import MyReservations from './components/MyReservations';
+import AddDoctor from './components/AddDoctor';
+import DeleteDoctor from './components/DeleteDoctor';
+import Root from './components/Root';
+import ReservationForm from './components/ReservationForm';
+
+const router = createBrowserRouter(
```

Comment:
- [ ] For some reason, I can't fetch the doctors, even after logging in:

![image](https://github.com/mailsg/medimate-frontend/assets/57421823/ec636de0-13ba-421d-9102-3911eb9593eb)

Kindly, make it so the doctors can be listed.  

This means "delete doctor" and reservations don't work either.  Kindly take a look.

If this is not a bug, this means your frontend app needs special extra procedures to be correctly run.  If that is the case, kindly add the specific instructions to your readme. 👍 

_This was tested both on Chrome and Firefox._